src/docs/arch-hs.dox

```diff
@@ -1,8 +1,125 @@
 /*! @arch_page arch-hs History Store
 
-The History Store in WiredTiger tracks old (all but the latest committed)
-versions of records.  By having these records in storage separate from
-the current version, they can be used to service long running transactions,
+The history store in WiredTiger tracks historical versions of records required to
+service older readers. By having these records in storage separate from
+the current version, they can be used to service long running transactions
 and be evicted as necessary, without interfering with activity that uses
-the most recent committed versions.
+the most recent committed versions. With the introduction of history store, only the
+newest update to a key is written to the user table while the older updates
+for the key are written to the history store. All user tables in the database are
+backed by a single history store table. The history store has no outward
```

Comment:
I think we want to be specific that for each update on the update chain a separate entry is created in the history store. I don't see a mention of that in the doc. If we could include an example update chain and the corresponding entries into the history store via a doxygen diagram then I think it'd be a lot clearer for readers.

Reply:
I have added an example update chain.